clients/cloud/python/producer.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Produce messages to Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+#
+# =============================================================================
+
+from confluent_kafka import Producer, KafkaError
+from confluent_kafka.admin import AdminClient, NewTopic
+import json
+import ccloud_lib
+
+
+
```

Comment:
`pip install flake8`, then run `flake8` from the directory containing these files to make sure they are formatted correctly.

Also verify these scripts with python3 (`virtualenv -p python3.6 ~/venv3 ; source ~/venv3/bin/activate`)

Reply:
@edenhill Done: reformatted with flake8 and verified python3.7